transformato/tests/test_charmm.py

```diff
@@ -22,11 +24,10 @@ def test_run_28_1h1q_rbfe_production_with_CHARMM():
     )
 
```

Comment:
@JohannesKarwou maybe we can come up with something better here? (1) I am not sure if we should really ship these tests, and (ii) we should really think about how to provide a good CHARMM experience. Maybe related: is there a conda binary for CHARMM?

Reply:
I totally agree with you, I'm also not that happy with these tests, but for now I have no better idea, but I sometimes need some tests, so currently I'm using them locally for me...
I don't know about a conda binary for CHARMM but I will ask Stefan. I just know that there is some pyCHARMM devolpment going on, maybe we can integrate that...